onnxruntime/test/python/test_pytorch_export_contrib_ops.py

```diff
@@ -109,6 +109,15 @@ def test_gelu(self):
         x = torch.randn(3, 3)
         self.run_test(model, x, custom_opsets={"com.microsoft": 1})
 
+    def test_gelu_supports_approximate_param(self):
+        # TODO(justinchu): Change this to a parameterized test
```

Comment:
Can you expand the comment to explain what's blocking this?

Reply:
We need parameterized as a dependency. Pytorch has it but ORT does not. Should we introduce the dependency at some point?